Engine/AI/NavMesh.cpp

```diff
@@ -706,26 +708,31 @@ bool NavMesh::FindPath(float3& start, float3& end, std::vector<float3>& path)
 
 void NavMesh::SaveNavMesh(unsigned char* nav_data, unsigned int nav_data_size) const
 {
-	std::string filepath(NAVMESH_PATH);
-	filepath.append("survival_scene_navmesh.bin");
+	Path* navmesh_path = App->filesystem->GetPath(RESOURCES_NAVMESH_PATH);
+	std::string navmesh_filename("survival_scene_navmesh.bin");
+
+	FileData navmesh_data{ reinterpret_cast<char*>(nav_data), nav_data_size };
 
-	App->filesystem->Save(filepath.c_str(), reinterpret_cast<char*>(nav_data), nav_data_size);
+	navmesh_path->Save(navmesh_filename.c_str(), navmesh_data);
```

Comment:
What would happen if I have multiple NavMesh? One for each scene at least, or as we said, a few for each scene.

Reply:
That will be implemented on the future but don't think is a problem.